RedHat-UX/red-hat-design-system
# docs: create minimal theme sheet for demos

## What I did

1. [x] add a minimal theme (base.css)
2. [x] add a demo stylesheet (demos.css)
3. [x] transform demos in 11ty so they load lightdom css from the correct path


## Testing Instructions

1. run dev server and 11ty build, and check deploy preview at https://deploy-preview-447--red-hat-design-system.netlify.app/components/footer/demo/global-footer/
